app/data/externalLinks/subHeaderLinks.ts

```diff
@@ -0,0 +1,14 @@
+const subHeaderLinks = [
+  {
+    name: "Email Us",
+    href: { pathname: "mailto:CleanBCIndustryFund@gov.bc.ca" },
+    highlightOn: ["/cif/operator(.*)"],
+  },
+  {
+    name: "Contact Information",
+    href: { pathname: "#" },
+    highlightOn: ["/cif/contact(.*)"],
```

Comment:
Possible to use an empty string (or some other kind of dummy value) here? Contacts is an existing internal page